### PR TITLE
feat: news item for build to python-build

### PR DESCRIPTION
**Pull request by beckermr**
PR Checklist:

- [x] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] if you are adding a new page under `docs/` or `community/`, you have added it to the sidebar in the corresponding `_sidebar.json` file
- [x] put any other relevant information below

xref: https://github.com/conda-forge/conda-forge.github.io/issues/2269

### CODE REVIEW

**Comment by beckermr** [comment]
pre-commit.ci autofix

**Comment by beckermr** [comment]
@conda-forge/core this is ready for review!

File: news/2024-08-21-remove-build.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Removing `build` in favor of `python-build`
+
+For nearly two years now, we have favored the use `python-build` as the package name for the PyPa `build`
```
Comment (jaimergp) [inline news/2024-08-21-remove-build.md:3]:
```suggestion
For nearly two years now, we have favored the use `python-build` as the package name for the PyPA [`build`](https://github.com/pypa/build)
```

File: news/2024-08-21-remove-build.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+project over `build`. In fact, the `build` package on `conda-forge` is archived and has not had its version
```
Comment (jaimergp) [inline news/2024-08-21-remove-build.md:4]:
```suggestion
project over `build`. In fact, the `build` feedstock on `conda-forge` is archived and has not had its version
```

**Pull request merged**